src/shared/helpers/is-user.ts

```diff
@@ -10,6 +12,20 @@ export function isUserElementaryPupil(user: CommonUserSchema) {
 	return isUser(user, SpecialUserGroup.PupilElementary);
 }
 
+export function isUserSecondaryTeacher(user: CommonUserSchema) {
+	return isUser(user, SpecialUserGroup.TeacherSecondary);
+}
+
+export function isUserDoubleTeacher(user: CommonUserSchema) {
```

Comment:
add some jsdoc what a double teacher is for someone who isn't familiar
i'm guessing teacher with secundairy and elementary education levels linked?

or maybe rename to:
```suggestion
export function isUserDoubleEducationLevelTeacher(user: CommonUserSchema) {
```